### DISCUSSION TITLE
Domain management issue (DNS)

### DISCUSSION THREAD

**Discussion by AhmedFathy2001** [vercel/community#6533] [Help]
### Question

I'm trying to move a domain i purchased this morning to vercel, I've moved 2 other domains to vercel right now to check if this issue persists, the domains are 4 days old and about 2 weeks old, along side the one i purchased this morning, it seems like when i try to move it to use vercel's DNS it gives me this error: Another Vercel account is using this domain. Set the following TXT record on _vercel.domain.com to use domain.com in this project. If you wish to [transfer](https://vercel.com/guides/how-can-i-move-a-domain-to-a-team#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.

I tried to resolve it many times, i added the TXT record then converted the dns but its not allowing me to manage it on vercel, the other domains have worked and are now managed by vercel so I'm not sure why it wont let this one work

very similar to: https://github.com/orgs/vercel/discussions/3488

**Comment by amyegan** [comment]
Hey @AhmedFathy2001. Can you walk me through the steps you have already tried?

**Reply by AhmedFathy2001** [reply]
Hey Amy, so here's what I've tried:
I went ahead and added the vercel TXT records as suggested in your previous response on the other post, then added the DNS records of vercel, then that work so i tried using the A records, that also didnt move the domain to my account currently it works, I have the dns still in namecheap but im trying to move it to vercel (the whole objective) is to make the domain vercel managed, but i keep getting that error message, so im not sure why is it used. as i said, I tried two other domains and they got moved to be managed by vercel but this one just wont work.

**Reply by AhmedFathy2001** [reply]
Hey @amyegan is there any update for me? Thanks
